src/logging.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Provides a wrapper for any logging service implementation of the expected
+ * logging service interface.
+ *
+ * This enables shared libraries or applications that want to support multiple
+ * logging service implementations to be coded against these wrapping functions,
+ * and allowing any concrete implementation to be injected.
```

Comment:
"and allowing" -> "and allows" or just "allowing" ?

Reply:
done